sql/plan/resolved_table.go

```diff
@@ -25,24 +25,43 @@ import (
 // ResolvedTable represents a resolved SQL Table.
 type ResolvedTable struct {
 	sql.Table
-	Database sql.Database
-	AsOf     interface{}
-	comment  string
+	SqlDatabase sql.Database
```

Comment:
Why this rename? Doesn't add anything imo

Reply:
The rename is because Go doesn't let you have a struct with a method and field with the same name. ResolvedTable needs to implement Databaser, which has a method named Database. So we have to rename the field.

I don't like it either.

Reply:
Fixed.